war/src/main/js/add-item.js

```diff
@@ -116,6 +116,18 @@ $.when(getItems()).done(function(data) {
       }
     }
 
+    function focusButton(status) {
+      var btn = $('form .footer .btn-decorator button[type=submit]');
```

Comment:
I got an error trying to use 
```
function getFormSubmitButton() {
    var btn = $('form .footer .btn-decorator button[type=submit]');
}
```
and then just calling it inside.
Do you think it is necessary to replace it?